code/modules/antagonists/protagonists/protagonist.dm

```diff
@@ -0,0 +1,39 @@
+/datum/antagonist/protagonist
+	name = "Protagonist"
+	roundend_category = "protagonists"
+	antagpanel_category = "Protagonists"
+	job_rank = ROLE_PROTAGONIST
+	show_to_ghosts = TRUE
+	///Some protagonists have min ages
+	var/min_age = 16
+	///Some protagonists have max ages
+	var/max_age = 99
+	///Outfit to put onto the character
+	var/outfit_type = /datum/outfit/wizard
```

Comment:
make this the naked outfit on purpose to signal someone forgetting to set this? probably not needed